lib/shoryuken/cli.rb

```diff
@@ -119,6 +119,10 @@ def parse_cli_args(argv)
           opts[:concurrency] = Integer(arg)
         end
 
+        o.on '-i', '--dispatch-interval DECIMAL', 'Dispatch interval' do |arg|
```

Comment:
Could you add a note what does this value represent, and what will change depending on this value?
Readme.md sounds like a good place

For instance, in case jobs are taking less than a second, let's say 200ms without this change; shoryuken will spend 800ms doing nothing; which ends up in doing just 20% of possible throughput.
On other side, this results in more calls to SQS, in case user is price sensitive